### PR TITLE
Dynamic input file - conf file changes to IO technology

### PR DESCRIPTION
**Pull request by manvanthar**
config file changes to work for all the IO devices, comments in PR#303 are included as part of this.

### CODE REVIEW

File: config/wrapper/pci_input.conf [hunk 1]
```diff
@@ -1,23 +1,92 @@
 [io_nvme_fvt]
+devices = interfaces:0
 disk = disks:0
+lv_disks = disks:0
+htx_disks = disks:1
 disks = disks:all
-device = interfaces:0
```
Comment (abdhaleegit) [inline config/wrapper/pci_input.conf:4]: we need this parameter for nvme functional test

File: config/wrapper/pci_input.conf [hunk 2]
```diff
@@ -1,23 +1,92 @@
...
 pci_device = functions:0
+pci_devices = functions:all
 module = driver
 
 [io_fc_fvt]
 disk = mpath_disks:0
-lv_disks = mpath_disks:1
-htx_disks = mpath_disks:2
+lv_disks = mpath_disks:0
```
Comment (abdhaleegit) [inline config/wrapper/pci_input.conf:13]: lvdisks ad disk should be different disk   
disk 0
lv_disks 1
htx_disks 2
disks all

File: config/wrapper/pci_input.conf [hunk 3]
```diff
@@ -1,23 +1,92 @@
...
+htx_disks = mpath_disks:1
 disks = mpath_disks:all
 wwids = mpath_wwids:all
 pci_device = functions:0
 pci_devices = functions:all
 module = driver
 
+[io_nvmf_rhel8_fvt]
+disk = mpath_disks:0
+lv_disks = mpath_disks:0
```
Comment (abdhaleegit) [inline config/wrapper/pci_input.conf:23]: same here  disk and lv_disks should be a diferent .. we follow this in CR for smooth runs... as disk will be used for raid/lv   with all tests variants...  lv_disks is used for pure lvm functionality test

File: config/wrapper/pci_input.conf [hunk 4]
```diff
@@ -1,23 +1,92 @@
...
+htx_disks = mpath_disks:1
+disks = mpath_disks:all
+wwids = mpath_wwids:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver:all
+
+[io_nvmf_fvt]
+device = disks:0
+disk = disks:0
```
Comment (abdhaleegit) [inline config/wrapper/pci_input.conf:33]: device is a nvme device and disk is a nvme space.. hope that is taken care

**Review by abdhaleegit — CHANGES_REQUESTED** [review]
keeping different devices as much as possible for different paramters would be good for smooth runs... we maintain the rule in CR inputfiles

File: config/wrapper/pci_input.conf (hunk 1 above)
Comment (manvanthar) [reply config/wrapper/pci_input.conf:4]: device = interface is still there Abdul point 2.

reason to reorder all the conf file was when we get the input file generated we have a standard way of data across devices and at the last we willhave only manually entered data for example passwordds

File: config/wrapper/pci_input.conf (hunk 4 above)
Comment (manvanthar) [reply config/wrapper/pci_input.conf:33]: yes that is taken care via input file.

**Comment by abdhaleegit** [comment]
LGTM

**Review by abdhaleegit — APPROVED** [review]
LGTM

**Comment by PraveenPenguin** [comment]
@manvanthar i feel it should in single commit , can you please rebase it

**Pull request closed without merging**